modules/webp-uploads/load.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * WebP-Uploads Module
+ *
+ * @package performance-lab
+ * @since 1.0.0
+ *
+ * Module Name: WebP uploads.
+ * Description: Uses WebP as the default format for new JPEG image uploads if the server supports it.
+ * Focus: images
+ * Experimental: No
+ */
+
+// Only enable if the server supports WebP.
+add_filter( 'image_editor_output_format', 'webp_default_filter_image_editor_output_format', 10, 3 );
```

Comment:
Nitpick, but can you move the filter right below the function as that is a more common practice? Also, more importantly, the comment above doesn't make sense anymore contextually, it needs to be moved above the check that is now within the function.

Reply:
👍🏼